main/main.cpp

```diff
@@ -284,7 +283,7 @@ void Main::print_help(const char *p_binary) {
 #ifdef TOOLS_ENABLED
 	OS::get_singleton()->print("  --export <target>                Export the project using the given export target. Export only main pack if path ends with .pck or .zip.\n");
 	OS::get_singleton()->print("  --export-debug <target>          Like --export, but use debug template.\n");
-	OS::get_singleton()->print("  --doctool <path>                 Dump the engine API reference to the given <path> in XML format, merging if existing files are found.\n");
+	OS::get_singleton()->print("  --doctool <path>                 Dump the engine API reference to the given <bool: inMarkdown> <path> in XML format, merging if existing files are found.\n");
```

Comment:
This shouldn't change `doctool`, just add a new option, e.g. `--dump-markdown-docs`.